src/main/resources/runner/environments/reg.env

```diff
@@ -1,4 +1,4 @@
-PF_VIR_MEM=50000
+PF_VIR_MEM=100000000
 PF_STK_MEM=8000
```

Comment:
Did you try bumping up the stack memory + max memory as well as virtual memory. That might have been why Stack was complaining.

Reply:
No, I am not sure about the command and I need to look it up before I make changes I am confident in.